src/EmployerProfiles/SFA.DAS.EmployerProfiles/Application/AccountUsers/Commands/UpsertEmployer/UpsertAccountCommandHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MediatR;
```

Comment:
I _think_ we can get rid of this handler now?

Reply:
Removed and pushed new changes